data/de/bwegt-efa.json

```diff
@@ -1,12 +1,84 @@
 {
     "name": "bwegt",
     "coverage": {
-        "realtimeCoverage": {
+        "regularCoverage": {
```

Comment:
I assume the change to "regular coverage" is intentional?

Reply:
Yes, that's what I tried to explain the commit message. bwegt produces nameless long distance trains while in the same area we have better alternatives available. The reduced coverage rating is meant to reflect that.

And yes, we probably want/need to review the coverage rating in general, the current system isn't really ideal for the automatic endpoint selection usecase. Therefore also #115, to give us better tools to inspect coverage overlaps and the ratings there.